static/js/wavelengthConverter.js

```diff
@@ -0,0 +1,63 @@
+const hex = (d) => Number(d).toString(16).padStart(2, "0");
+const wavelengthToHex = (wavelength) => {
+  const Gamma = 0.8;
```

Comment:
If anything, I think the wavelength divide by ten should happen at the entrance to this function (make sure to comment that this is converting angstrom to nm.